test/power_divergence.jl

```diff
@@ -26,6 +26,7 @@ for i = 1:length(c)
 end
 
 @test pvalue(m) ≈ 2.9535891832117357e-7
+@test "$(StatsBase.PValue(pvalue(m)))" == "<1e-6"
```

Comment:
The point isn't to test StatsBase, it's to test the result of `sprint(show, m)`. That should allow you to test both the p-value and the confidence interval at the same time. Use triple-quoted strings to hold the expected output. BTW, I think this should go to a separate file, e.g. `show.jl`, since there's no point in testing the output for all types as long as the code is common.

Reply:
Sorry, I didn't know how to get the output from show. Using `sprint(show, m)` dumps the whole output into a string. What is the most elegant way of extracting, e.g., the confidence interval? I don't know what you mean by "Use triple-quoted strings to hold the expected output". Could you give me a link to an example?

Reply:
Ok, I think I have now understood what you meant and included three tests in a new file show.jl.

One last question: I tried including line breaks in the output of `sprint(show, m)` but then the tests would fail. Is there a way of not having all of the output in one enormous line?

Reply:
You can just introduce normal line breaks, just like in a copy/paste of the output. Any indentation will automatically be removed by comparing with the position of triple quotes on the first line.